index.html

```diff
@@ -47,8 +47,189 @@ <h1 class="h1">I'm Samson<br>Glad to see<br>you!</h1>
+                </div>
+                <div class="text-box">
+                    <h2>Tonic</h2>
+                    <div class="canopy">
+                        <h4>CANOPY</h4>
+                        <ul class="back">
+                            <li><img class="dot" src="images/Counter.png" alt="dot">Back End Dev</li>
+                            <li><img class="dot" src="images/Counter.png" alt="dot">2015</li>
+                        </ul>
+                    </div>
+                    <div>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <div class="small-btn-div">
+                        <div class="small-btn"><a>html</a></div>
+                        <div class="small-btn"><a>css</a></div>
+                        <div class="small-btn"><a>javaScript</a></div>
+                    </div>
+                    <button><a>See Project</a></button>
+                </div>
+            </div>
+            <div class="grid-item grid-item-4">
+                <div class="card-box">
+                    <img src="images/Snapshoot Portfolio4.png" alt="" srcset="">
+                </div>
+                <div class="text-box">
+                    <h2>Multi-Post<br>Stories</h2>
+                    <div class="canopy">
+                        <h4>CANOPY</h4>
+                        <ul class="back">
+                            <li><img class="dot" src="images/Counter.png" alt="dot">Back End Dev</li>
+                            <li><img class="dot" src="images/Counter.png" alt="dot">2015</li>
+                        </ul>
+                    </div>
+                    <div>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <div class="small-btn-div">
+                        <div class="small-btn"><a>html</a></div>
+                        <div class="small-btn"><a>css</a></div>
+                        <div class="small-btn"><a>javaScript</a></div>
+                    </div>
+                    <button><a>See Project</a></button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <div class="grid-container">
+            <div class="footer-text">
+                <h2>About<br>Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div class="con">
+                    <a class="connect">
+                      LET'S CONNECT
+                    </a>
```

Comment:
:red_circle: I think the right semantic tag here is hX tag (heading) than a tag link. A heading is a short phrase that indicates what the next section is about.